arm/Microsoft.Automation/automationAccounts/.parameters/encr.parameters.json

```diff
@@ -5,22 +5,21 @@
         "name": {
             "value": "<<namePrefix>>-wd-aut-encr-001"
         },
-        "encryptionKeySource" : {
-            "value" : "Microsoft.Keyvault"
+        "encryptionKeySource": {
```

Comment:
There's no reference of the automation account changes in the PR title and description. Are these changes intended to be included here?

Reply:
The fix for the indent? Should I create a new PR for that? It was an automatic fix when I saved the file.

Reply:
I think just the incorrect 'carml' in the keyvaulturi was an actually fix

Reply:
By the way, the reason I even touched the automation account in the first place was that I used its CMK implementation as inspiration ;)

Reply:
Again I'm not discussing the value of it but generally kindly asking to make the review process easier. Nothing is mentioned in the PR description about this change so I'm not expecting the automation account to have updates. Could you please add one line in the PR description?

Reply:
Removed